### PR TITLE
Add ReflectHelper class to better facilitate various reflection actions.

### PR DESCRIPTION
**Pull request by benwoo1110**
Various methods to help facility reflection.

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiverseCore/utils/ReflectHelper.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+     * @return A {@link Class} if found, else null.
+     */
+    @Nullable
+    public static Class<?> getClass(String classPath) {
+        try {
+            return Class.forName(classPath);
+        } catch (ClassNotFoundException e) {
+            return null;
+        }
+    }
```
Comment (dumptruckman) [inline src/main/java/com/onarandombox/MultiverseCore/utils/ReflectHelper.java:26]: Hopefully we're not using this all the time as it is a relatively costly operation. Class.forName is "slow" and should an exception be thrown that will also slow this down quite a bit.

Comment (dumptruckman) [reply src/main/java/com/onarandombox/MultiverseCore/utils/ReflectHelper.java:26]: We might want to consider a caching strategy here.

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/utils/ReflectHelper.java:26]: Yea we do store the result, it should only be used on started, then the results are stored.

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/utils/ReflectHelper.java:26]: You can see #2580

**Review by dumptruckman — APPROVED** [review]
ok for now.

**Review by nicegamer7 — COMMENTED** [review]
Seems good I guess 👍.

**Pull request merged**